### ISSUE TITLE
Using DSN with SQLDriverConnect does not use the DSN settings

### ISSUE THREAD

**Issue by Wonixen** [duckdb/duckdb-odbc#48]
I have a DSN MyDuckDb where I have 4 keys:
database                                            -> "c:\dataPath\myDuckDb.ddb"
driver                                                 -> "C:\Windows\System32\duckdb_odbc.dll"
debug_window_mode                       -> "window"
debug_skip_checkpoint_on_commit -> "yes"

when using SQLDriverConnect I use: "DSN=MyDuckDb", but it always connect to :memory:, and does not take into account the other two settings: debug_window_mode and debug_skip_checkpoint_on_commit.

**Comment by staticlibs** [comment]
I think this is a duplicate of #29.

**Comment by Wonixen** [comment]
Yes and no, like #29 it does not use the database name, but it does not read the other settings. so the problem is bigger that what #29 seems to implied.
Maybe #29 could be modified to also mention that any setting in ODBC.ini are ignored!

**Comment by staticlibs** [comment]
Yes, this is correct for #29 too, if the connection string ends with a semicolon (that is always true on Windows), then the DSN value is not set internally and DSN configuration from registry (`HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBC.INI`) is not loaded at all. So this is likely to be the same problem as #29.

**Comment by staticlibs** [comment]
@Wonixen 

I wonder if you can try the [latest nightly](https://duckdb.org/docs/installation/?version=main&environment=odbc&platform=win&download_method=direct&architecture=x86_64) (needs to be manually switched to Windows, link points to Linux version)? I believe this was fixed by #53 and #60.

**Comment by staticlibs** [comment]
Closing this as completed in `v1.2.2`, please re-open if necessary.
